pyproject.toml

```diff
@@ -105,19 +103,21 @@ sqlparse = "0.3.1"
 tablib = "2.0.0"
 terminaltables = "3.1.0"
 toml = "0.10.1"
-typed-ast = "1.4.1"
 ujson = "3.2.0"
 Unidecode = "1.1.1"
 urllib3 = "1.26.5"
 URLObject = "2.4.3"
-uWSGI = "2.0.19.1"
+uWSGI = "2.0.23"
 webencodings = "0.5.1"
 wrapt = "1.12.1"
 yapf = "0.30.0"
 YURL = "1.0.0"
 zipp = "3.1.0"
 ipython = "^8.4.0"
 rt = "^2.2"
+django-recaptcha = "3.0.0"
+psycopg = "^3.1.17"
+pymemcache = "^4.0.0"
```

Comment:
Sites should configure the memcached setting in their custom settings, alongside the database, since it will cause errors if memcached isn't in the docker-compose stack.